### PR TITLE
remove session secret

### PR DESCRIPTION
**Pull request by JoeChapman**
remove the session secret so our secrets (intended for the server) are not published

### CODE REVIEW

**Comment by sulthan-ahmed** [comment]
So when we run the app locally, do we just insert a secret? The app requires a secret so it seems you can't just now clone the repo and run the app to get it working straight out of the box

**Comment by JoeChapman** [comment]
Boot strap sets a default https://github.com/UKHomeOfficeForms/hof-bootstrap/blob/master/lib/defaults.js#L23

File: hof.settings.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
     "./apps/right-to-rent-check"
   ],
   "session": {
-    "name": "right-to-rent-check.hof.sid",
-    "secret": "0442ba333a3c4b514311af1d1ee78827cced4d5e9fa307010eff23e4e40d4a48f474b8881bff60467ae041fce749a8d19d2d2bc060c0722b4cc9a2457671f208"
+    "name": "right-to-rent-check.hof.sid"
   }
```
Comment (sulthan-ahmed) [inline hof.settings.json]: I just showed @lennym and he agreed that is going to cause a problem.  He said to just remove the session property. This seems to have resolved the problem and now the app runs properly.  

The session property here seems to not merge with the bootstrap property but overwrite it.

Comment (JoeChapman) [inline hof.settings.json]: ok

**Pull request merged**